RoaringBitmap/src/test/java/org/roaringbitmap/buffer/TestBitSetUtil.java

```diff
@@ -234,4 +235,10 @@ private static ByteBuffer toByteBuffer(BitSet bitset) {
     return ByteBuffer.wrap(bitset.toByteArray());
   }
 
+  @Test
+  public void bitSetOf() {
+    ImmutableRoaringBitmap bitmap = ImmutableRoaringBitmap.bitmapOf(1, 2, 4);
```

Comment:
Please add a test with something like...
```
ImmutableRoaringBitmap bitmap = ImmutableRoaringBitmap.bitmapOf(1, 2, 4, -1);
```

Reply:
I did not realize this basic fact, see [my proposed solution](https://github.com/RoaringBitmap/RoaringBitmap/pull/661/commits/56e8e6a10c9cd98f556b41154e34e7a9a34a8aac) - method will possibly return two bit sets, the second one for "negative" values shifted by Integer.MIN_VALUE not to be negative and can be hold by bit set. For obvious reasons bit set holding positive values including zero is returned always even being empty.

Reply:
[Done](https://github.com/RoaringBitmap/RoaringBitmap/commit/372c0b551716c33884b373c42a3112690b2801ca#diff-3eb787aff967a9a86a3c2fa49ff908ee1e1caf516b2002f9e5e6f53a05adcc14).